chroma-manager/tests/integration/core/chroma_integration_testcase.py

```diff
@@ -425,12 +425,13 @@ def create_filesystem(self, hosts, filesystem, verify_successful = True):
         filesystem_id = response.json['filesystem']['id']
         command_id = response.json['command']['id']
 
-        self.wait_for_command(
-            self.chroma_manager,
-            command_id,
-            verify_successful=verify_successful,
-            timeout = LONG_TEST_TIMEOUT
-        )
+        self._fetch_help(lambda: self.wait_for_command(self.chroma_manager,
```

Comment:
Does this still need to be here?

Reply:
removed but may need to add again for further debug